packages/web-components/src/components/card/card.ts

```diff
@@ -140,6 +190,12 @@ class DDSCard extends StableSelectorMixin(DDSLink) {
   @property()
   href = '';
 
+  /**
+   * Pictogram position
+   */
+  @property({ attribute: 'pictogram-position', reflect: true })
+  pictogramPosition = PICTOGRAM_POSITION.TOP;
```

Comment:
To align to `iconPlacement` for `<dds-card-footer>`:

```suggestion
  pictogramPlacement = PICTOGRAM_POSITION.TOP;
```